tests/integration/synapseclient/test_json_schema_services.py

```diff
@@ -63,13 +63,13 @@ def test_json_schema_organization(js):
 
 
 class TestJsonSchemaSchemas:
-    @pytest.fixture(autouse=True)
+    @pytest.fixture(autouse=True, scope="function")
```

Comment:
This is an effort to resolve these integration test errors:

> Semantic version: '0.48786.1' already exists for this JSON schema

That have been popping up. These passed on my local machine - Will be verifying they also pass in the CI pipeline

Reply:
Can confirm these tests also passed on my machine.